smoketests/tests/csharp_module.py

```diff
@@ -19,7 +19,7 @@ def test_build_csharp_module(self):
         try:
 
             run_cmd("dotnet", "nuget", "locals", "all", "--clear", cwd=bindings, capture_stderr=True)
-            run_cmd("dotnet", "workload", "install", "wasi-experimental", "--skip-manifest-update")
+            run_cmd("dotnet", "workload", "install", "wasi-experimental", "--skip-manifest-update", cwd=STDB_DIR / "modules")
```

Comment:
was this only needed on windows or something? o_O

Reply:
something to do with dotnet sdk version selection; the windows-latest runner has v9 installed but our runner doesn't, I think.